stixcore/products/level2/housekeepingL2.py

```diff
@@ -342,6 +342,18 @@ def __init__(self, *, service_type=0, service_subtype=0, ssid=1, control, data,
         self.service_subtype = 0
         self.service_type = 0
 
+    @property
+    def dmin(self):
+        return min([self.data['y_srf'].min().value, self.data['z_srf'].min().value])
```

Comment:
Better to use `to_value('km')` incase the `value` is in different units, not the case here but doesn't hurt.